plugin/build.gradle

```diff
@@ -375,16 +375,18 @@ List<String> jacocoExclusions = [
         'org.opensearch.ml.jobs.MLBatchTaskUpdateExtension',
         'org.opensearch.ml.jobs.MLBatchTaskUpdateJobRunner',
         'org.opensearch.ml.action.agents.TransportRegisterAgentAction',
-        'org.opensearch.ml.action.mcpserver.TransportMcpToolsRemoveOnNodesAction',
+        'org.opensearch.ml.action.mcpserver.TransportMcpToolsListAction.1',
```

Comment:
Can we add an issue to add tests for these files? This list keeps growing :)

Reply:
Done: https://github.com/opensearch-project/ml-commons/issues/3895